tests/Feature/BlueprintTest.php

```diff
@@ -117,6 +117,22 @@ public function it_parses_shorthands(): void
         ], $this->subject->parse($blueprint));
     }
 
+    #[Test]
+    public function it_parses_ulid_shorthand(): void
+    {
+        $blueprint = $this->fixture('drafts/ulid-shorthand.yaml');
+
+        $this->assertEquals([
+            'models' => [
+                'Person' => [
+                    'id' => 'ulid primary',
```

Comment:
It should not be required to say `ulid primary`, just `ulid` is fine as it is the conventional `id` field.

Reply:
It isn't required to specify `ulid primary`. Here is the [draft](https://github.com/laravel-shift/blueprint/pull/657/files#diff-20e2cd8edfa713aaf638d0916fbda107837b16fd178030ca9c8b936a04ef4283) file that was used in that test `tests/fixtures/drafts/ulid-shorthand.yaml`

Reply:
Since it is not needed, please remove `ulid primary` from all of your new fixtures to avoid confusion for those reviewing them as examples.

Reply:
Fixed.

Reply:
Fixed.